src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java

```diff
@@ -65,95 +64,133 @@ private SecurityDataResult processTestHelper( String cve_id) throws Exception
         return sdpr;
     }
 
-
-    @Test
-    public void validLookUpTest() throws Exception
-    {
-            
-        String cve_id = "2016-6346";
-
-        SecurityDataProcessor sdp = new SecurityDataProcessor("anycpe");
-
-        SecurityDataJSON json = (SecurityDataJSON)executeMethod( sdp, "lookUpAPI", new Object [] { cve_id });
-
-        logger.info("ID {}", json.getName().toString());
-
-        logger.info("package_state {}", json.getPackageState().get(0).getCpe());
-      
-        assertTrue( true );
-    }
-
     @Test
     public void invalidLookUpTest() throws Exception
     {
-            
-        String cve_id = "2016-63461";
+        Boolean catched = false;
+        String cve_id = "2016-634611";
 
         SecurityDataProcessor sdp = new SecurityDataProcessor("anycpe");
 
         try {
             SecurityDataJSON json = (SecurityDataJSON)executeMethod( sdp, "lookUpAPI", new Object [] { cve_id });
             logger.info("json {}", json.toString());
         } catch (FileNotFoundException e)  {
-            logger.info("catched exception. {}", e.toString());
+            catched = true;
         }
-        assertTrue( true );
+        
+        assertTrue( catched );
     }
 
     
     @Test
     public void failByNoCPETest() throws Exception
     {
         // this test won't be able to use anymore, once CVE-2016-6346 security data is updated 
+        String cpe="arbitrary_cpe";
         String cve_id = "2016-6346";
 
-        SecurityDataResult sdpr = processTestHelper( cve_id);
+        SecurityDataResult sdpr = processTestHelper(cpe, cve_id);
 
         sdpr.getFail();
```

Comment:
I think this line is redundant isn't it? (and same applies to similar calls below)